tools/travis_script.py

```diff
@@ -135,10 +135,15 @@ def build_iotjs(buildtype, args=[], env=[]):
         for buildtype in BUILDTYPES:
             if buildtype == 'release':
                 set_release_config_tizenrt()
+            # Fixme: EXTRA_LIBPATHS and EXTRA_LIB can be deleted
+            # when TizenRT uses jerry-ext
             exec_docker(DOCKER_TIZENRT_OS_PATH, [
                         'make', 'IOTJS_ROOT_DIR=' + DOCKER_IOTJS_PATH,
                         'IOTJS_BUILD_OPTION='
-                        '--profile=test/profiles/tizenrt.profile'])
+                        '--profile=test/profiles/tizenrt.profile',
+                        'EXTRA_LIBPATHS=-L' + DOCKER_IOTJS_PATH +
+                        '/build/arm-tizenrt/' + buildtype + '/lib/',
+                        'EXTRA_LIBS=-ljerry-ext'])
```

Comment:
Do you really need to provide the -L / -l option flags in the `EXTRA_LIBPATHS` and `EXTRA_LIBS` values?

Reply:
No need to answer, I see you need to give it, but I was surprised at first.